droughtpipeline/pipeline.py

```diff
@@ -75,6 +78,12 @@ def run_pipeline(
         if send:
             if not forecast:
                 logging.info("get drought forecasts from storage")
+                self.data.forecast_climateregion = self.load.get_pipeline_data(
```

Comment:
Is it possible to split `get_pipeline_data` into separate functions? It appears to return `forecast_climateregion` and `forecast_admin` based on input parameters. Maybe there are good reasons to keep this in the same function but let's see if we can split it into simpler and smaller functions.